test/functional/Valhalla/src_lw5/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -83,6 +90,11 @@ static value class Point2D {
 		Point2D(int[] position) {
 			this(position[0], position[1]);
 		}
+
+		void checkEqualPoint2D(int[] position) throws Throwable {
```

Comment:
No, I just thought it made more sense to have this method as part of Point2D instead of as an external helper method. Previously it was done this way since the class was fully generated by asm.